acra-core/src/main/java/org/acra/config/ConfigUtils.java

```diff
@@ -28,7 +29,9 @@
 
     @NonNull
     public static <T extends Configuration> T getPluginConfiguration(@NonNull CoreConfiguration config, @NonNull Class<T> c) {
+        if (ACRA.DEV_LOGGING) ACRA.log.d(ACRA.LOG_TAG, "Checking plugin Configurations : " + config.pluginConfigurations() + " for class : " + c);
         for (Configuration configuration : config.pluginConfigurations()) {
+            if (ACRA.DEV_LOGGING) ACRA.log.d(ACRA.LOG_TAG, "Checking plugin Configuration : " + configuration + " against plugin class : " + c);
```

Comment:
Printing this for each class is kind of duplicated and will flood the log

Reply:
It doesn't flood the log. It prints 2 or 3 lines.
And it gives you information about how far through the iterator it got before exiting.